Wrappers/Python/cil/recon/FBP.py

```diff
@@ -319,11 +319,18 @@ def _pre_filtering(self,acquistion_data):
         weights_ptr = self._weights.ctypes.data_as(c_float_p)
 
         ag = acquistion_data.geometry
-        if ag.dimension_labels == ('angle','vertical','horizontal'):
+
+        if ag.geom_type & AcquisitionType.CONE_FLEX:
+            projection_label = 'projection'
+        else:
+            projection_label = 'angle'
+
+
+        if ag.dimension_labels == (projection_label,'vertical','horizontal'):
             cilacc.filter_projections_avh(data_ptr, filter_ptr, weights_ptr, self.fft_order, *acquistion_data.shape)
-        elif ag.dimension_labels == ('vertical','angle','horizontal'):
+        elif ag.dimension_labels == ('vertical',projection_label,'horizontal'):
             cilacc.filter_projections_vah(data_ptr, filter_ptr, weights_ptr, self.fft_order, *acquistion_data.shape)
-        elif ag.dimension_labels == ('angle','horizontal'):
+        elif ag.dimension_labels == (projection_label,'horizontal'):
```

Comment:
This is going ahead of the implementation. We would not get to this code block with the flex geometry with astra anyway.

Move to a follow up PR when we add flex geometry support with TIGRE.

Reply:
done